src/js/legend_config.js

```diff
@@ -40,6 +41,10 @@ export const sections = [
     name: "Railroads",
     entries: RailLayers.legendEntries,
   },
+  {
+    name: "Aerialways",
```

Comment:
I think “aerialway” is only a real word in OSM English. Both aerial trams and chair lifts count as _aerial lifts_.

Reply:
Chairlifts or gondolas on the east coast.

Reply:
Tram sounds like some quaint en_GB thing to me.

Reply:
_Aerial lift_ sounds good to me as a general term.

The two transit-oriented aerial lifts I know of in the United States have _tram_ in their names:
* [Portland Aerial Tram](https://en.wikipedia.org/wiki/Portland_Aerial_Tram)
* [Roosevelt Island Tramway](https://en.wikipedia.org/wiki/Roosevelt_Island_Tramway)

I think "aerial tramway" in US English is reasonably understood to be a transit-oriented `aerialway=gondola` or `aerialway=cable_car`. But at a ski resort, I'd definitely call that a gondola.